README.md

```diff
@@ -145,15 +145,131 @@ chef_ingredient 'analytics' do
 end
 ```
 
+## Chef Infrastructure Resources
+
+### General Properties
+
+These properties exist for all infrastructure resources
+
+- `name` - A name for the resource.
+- `channel` - The channel from our package repository to install. Most of the time you want stable.
+- `version` - The version of Automate you want to install.
+- `config` - The configuration that will be written to the appropriate configuration file for the product.
+- `accept_license` - Do you accept Chef's license agreements.
+- `platform` - Use only if you need to over-ride the default platform.
+- `platform_version` - Use only if you need to over-ride the default platform.
+
+### chef_automate
+
+Installs Chef Automate.
+
+#### Properties
+- `enterprise` - The Enterprise to create in Automate
+- `license` - we recommend using the chef_file resource
+- `chef_user` - The user you will connect to the Chef server as
+- `chef_user_pem` - The private key of the above Chef user
+- `validation_pem` - The validator key of the Chef org we're connecting to
+- `builder_pem` - The private key of the build nodes
+
+### chef_backend
+
+#### Properties
+
+- `bootstrap_node` - The node we'll bootstrap secrets with.
```

Comment:
This is actually `:peers` now, and can accept an array of peers.  If it can talk to any of them, it will join the cluster.  If it cannot, it assumes it is the bootstrap, and starts a new cluster.